nixos/modules/services/misc/atuin.nix

```
@@ -85,7 +87,7 @@ in
       wantedBy = [ "multi-user.target" ];
 
       serviceConfig = {
-        ExecStart = "${pkgs.atuin}/bin/atuin server start";
+        ExecStart = "${cfg.package}/bin/atuin server start";
```

```suggestion
        ExecStart = "${lig.getExe cfg.package} server start";
```
this makes it easier for people overwriting the package to control what binary is started.

---

> this makes it easier for people overwriting the package to control what binary is started.

Thanks for that suggestion! That's a much better way of doing it.

---

P.s. the reason I went with `${cfg.package}/bin/atuin` was because all the service modules I came across seemed to use that. Doing a rough `rg` count, `lib.getExe cfg.package` only occurs 68 times in nixpkgs, but `{cfg.package}/bin` occurs 470 times (total occurrences not files). I don't know if this is a potential target for a tree-wide replacement?

---

> I don't know if this is a potential target for a tree-wide replacement?

Could be but it could cause breakages. Not sure if we can easily proof, that the change is a noop.